extensions/amp-access/0.1/access-expr-impl.jison

```diff
@@ -0,0 +1,130 @@
+
+/* Lexical grammar */
+%lex
+%%
+\s+                       /* skip whitespace */
+"AND"                     return 'AND'
+"OR"                      return 'OR'
+"NOT"                     return 'NOT'
+"NULL"                    return 'NULL'
+"TRUE"                    return 'TRUE'
+"true"                    return 'TRUE'
+"FALSE"                   return 'FALSE'
+"false"                   return 'FALSE'
+"("                       return '('
+")"                       return ')'
+"|"                       return '|'
+"<="                      return 'LTE'
+"<"                       return 'LT'
+">="                      return 'GTE'
+">"                       return 'GT'
+"!="                      return 'NEQ'
+"=="                      return 'EQ'
+"="                       return 'EQ'
```

Comment:
What do you think? It's loosely based on SQL where there's no `==`. Should I remove then?